source/cloud-security/gcp/prerequisites/pubsub.rst

```diff
@@ -17,7 +17,15 @@ In this section, we see how to create a topic, a subscription, and a sink to ful
 Create a topic
 --------------
 
-Every publishing application sends messages to topics. Wazuh will retrieve the logs from this topic.
+Every publishing application sends messages to topics. Wazuh will retrieve the logs from this topic. To create it, follow these steps:
+
+#. Go to the **Pub/Sub** service.
+#. Click on **CREATE TOPIC**.
```

Comment:
The "Create topic" button is on the previous window, I didn't add an image because I thought it wasn't necessary for that single step. Should we add it?

![topics](https://github.com/wazuh/wazuh-documentation/assets/51374959/834f30f8-8b14-497a-ae5e-38b769eefd8d)

Reply:
I was just making sure we were using the right reference name.